FindChannelFunction/src/test/java/no/unit/nva/channel/ChannelRegistryClientTest.java

```diff
@@ -0,0 +1,68 @@
+package no.unit.nva.channel;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import no.unit.nva.channel.exception.NoResultsFoundException;
+import no.unit.nva.channel.model.outgoing.Channel;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class ChannelRegistryClientTest {
+
+    private ObjectMapper objectMapper = FindChannelFunctionApp.createObjectMapper();
+
+    @Test(expected = NoResultsFoundException.class)
```

Comment:
I started using Jupiter in another project, this allows better methods for testing exceptions